CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+### 2.27.0
+This release includes:
+* An Amazon Linux 2 Base
+* Fluent Bit [1.9.6](https://fluentbit.io/announcements/v1.9.6/)
+* Amazon CloudWatch Logs for Fluent Bit 1.9.0
+* Amazon Kinesis Streams for Fluent Bit 1.10.0
+* Amazon Kinesis Firehose for Fluent Bit 1.7.0
+
+Compared to `2.26.0` this release adds:
+* Feature - Add support for building `cloudwatch` Go plugin on Windows [cloudwatch:269](https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit/pull/269)
+* Feature - Add support for building `kinesis` Go plugin on Windows [kinesis:230](https://github.com/aws/amazon-kinesis-streams-for-fluent-bit/pull/230)
+* Feature - Add support for building `firehose` Go plugin on Windows [firehose:200](https://github.com/aws/amazon-kinesis-firehose-for-fluent-bit/pull/200)
```

Comment:
I thought we agreed in chat that we were not mentioning these in the AWS for Fluent Bit release notes since they have no value for customers here. The change only affects those repos. This will confuse folks I fear...

Reply:
I see. I will delete those